com.unity.render-pipelines.high-definition/Editor/PostProcessing/ExposureEditor.cs

```diff
@@ -204,18 +204,18 @@ public override void OnInspectorGUI()
             }
         }
 
-        // TODO: See if it's possible to refactor into a custom VolumeParameterDrawer
-        void DoFixedExposureField(SerializedDataParameter fixedExposure)
+        // TODO: See if this can be refactored into a custom VolumeParameterDrawer
```

Comment:
Could be good to actually do it. This should be doable AFAIK. It'll let user the possibility to use this in their volume inspectors. Here it is not usable to users.